### ISSUE TITLE
ERROR in ./src/entry.js when running demo

### ISSUE THREAD

**Issue by jumasheff** [mderrick/react-html5video#30]
```
ERROR in ./src/entry.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./components/index/index in /home/user/projects/react-html5video/demo/src
 @ ./src/entry.js 15:28-63
```
There are many similar errors, caused by import statements, where path is written with lowercase letters, while actual filenames begin with Uppercase: i.e. `components/index/index` vs `components/index/Index`.

node --version: 6.0.0
npm --version: 3.8.6

After I've edited all the import statements the errors are gone.
Should I send pull request?

**Comment by mderrick** [comment]
Ah, sure a PR would be great thanks! What OS are you on?

**Comment by muratjumashev** [comment]
Ubuntu 14.04
I found out that in OSX imports are case-insensitive, but in Linux they are.
http://stackoverflow.com/questions/13617863/can-i-force-node-js-require-to-be-case-sensitive
https://mail.mozilla.org/pipermail/es-discuss/2013-April/030165.html

**Comment by mderrick** [comment]
Yea I thought that was the case. My CI build usually catches this but I've not added any tests to this.....yet.

**Comment by mderrick** [comment]
Thanks for your help. Published in 1.2.9 :)
